documentation/book/proc-deploying-the-topic-operator-standalone.adoc

```diff
@@ -0,0 +1,63 @@
+
+[id='deploying-the-topic-operator-standalone']
+= Deploying the Topic Operator standalone
+
+Deploying the Topic Operator as a standalone component is more complicated than installing it using the Cluster Operator, but is more flexible.
+For instance is can operate _with_ any Kafka cluster, not necessarily one deployed by the Cluster Operator.
+
+.Prerequisites
+
+* An existing Kafka cluster for the Topic Operator to connect to.
+
+.Procedure
+
+. Edit the `examples/install/cluster-operator/05-Deployment-strimzi-topic-operator.yaml` resource. You will need to change the following
++
+* The `STRIMZI_KAFKA_BOOTSTRAP_SERVERS` environment variable in `Deployment.spec.template.spec.containers[0].env` should be set to a list of bootstrap brokers in your Kafka cluster, given as a comma-separated list of `_<hostname>_:‍_<port>_` pairs.
```

Comment:
These should be two dots instead of stars. So replace * with ..

Reply:
This got skipped.

Reply:
Sorry I missed that. THB I felt bullets looked better than an ordered sublist. This there some rule for deciding which should be used? It's not like the order in which these things get changes matters to the correctness of the procedure.